azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -46,12 +50,18 @@
 
   public static final String COMMAND = "command";
   public static final String AZKABAN_MEMORY_CHECK = "azkaban.memory.check";
+  // Use AZKABAN_SERVER_NATIVE_LIB_FOLDER Configuration Key instead
```

Comment:
nit: Can mention the full reference:
`azkaban.Constants.ConfigurationKeys.AZKABAN_SERVER_NATIVE_LIB_FOLDER`

This helps IntelliJ track renames inside comments